userguide/one_shot_object_detection/README.md

```diff
@@ -89,11 +130,11 @@ tc.one_shot_object_detector.util.preview_synthetic_training_data(starter_images,
 synthetic_images.explore()
 ```
 
-![Explore synthetic images](images/synthetic_images_explore.jpg)
+![Explore synthetic images](images/synthetic_images_explore.png)
```

Comment:
Is there a reason for changing the file format from `jpg` to a heavier format `png`?

If the image files have been changed, I would recommend compressing them. For example the image below is 2.95MB now vs 916KB before.

Reply:
Done.